translations/kubectl/pt_BR/LC_MESSAGES/k8s.po

```diff
@@ -0,0 +1,3319 @@
+
+#: pkg/kubectl/cmd/set/set_selector.go:82
+msgid "Set the selector on a resource"
+msgstr "Define um seletor em um recurso"
+
+#: pkg/kubectl/cmd/config/create_cluster.go:68
+msgid "Sets a cluster entry in kubeconfig"
+msgstr "Define um cluster no arquivo kubeconfig"
+
+#: pkg/kubectl/cmd/config/create_context.go:58
+msgid "Sets a context entry in kubeconfig"
+msgstr "Define um context no arquivo kubeconfig"
+
+#: pkg/kubectl/cmd/config/create_authinfo.go:104
+msgid "Sets a user entry in kubeconfig"
+msgstr "Define um usuário no arquivo kubeconfig"
+
+#: pkg/kubectl/cmd/config/set.go:60
+msgid "Sets an individual value in a kubeconfig file"
+msgstr "Define um valor individual no arquivo kubeconfig"
+
+#: pkg/kubectl/cmd/config/use_context.go:49
+msgid "Sets the current-context in a kubeconfig file"
+msgstr "Define o current-context no arquivo kubeconfig"
+
+#: pkg/kubectl/cmd/describe.go:86
+msgid "Show details of a specific resource or group of resources"
+msgstr "Mostra os detalhes de um recurso específico ou de um grupo de recursos"
+
+#: pkg/kubectl/cmd/rollout/rollout_status.go:58
+msgid "Show the status of the rollout"
+msgstr "Mostra o status de um rollout"
+
+#: pkg/kubectl/cmd/expose.go:106
+msgid "Synonym for --target-port"
+msgstr "Sinônimo para —target-port"
+
+#: pkg/kubectl/cmd/expose.go:88
+msgid ""
+"Take a replication controller, service, deployment or pod and expose it as a "
+"new Kubernetes Service"
+msgstr ""
+"Pega um replication controlar, service, deployment ou por e expõe como um "
+"novo Serviço do Kubernetes"
+
+#: pkg/kubectl/cmd/run.go:117
+msgid "The image for the container to run."
+msgstr "A imagem para o container executar."
+
+#: pkg/kubectl/cmd/run.go:119
+msgid ""
+"The image pull policy for the container. If left empty, this value will not "
+"be specified by the client and defaulted by the server"
+msgstr ""
+"A política de pula de imagens. Se deixado em branco, este valor não será "
+"especificado pelo cliente se padronizado pelo server"
+
+#: pkg/kubectl/cmd/rollingupdate.go:101
+msgid ""
+"The key to use to differentiate between two different controllers, default "
+"'deployment'.  Only relevant when --image is specified, ignored otherwise"
+msgstr ""
+"A chave utilizada para diferenciar entre dois controlares diferentes, padrão "
+"'deployment'.  Apenas relevante quando --image é especificado, é ignorado "
+"caso contrário"
+
+#: pkg/kubectl/cmd/create_pdb.go:63
+msgid ""
+"The minimum number or percentage of available pods this budget requires."
+msgstr ""
+"Um número mínimo ou porcentagem de pods disponíveis que este budget requer."
+
+#: pkg/kubectl/cmd/expose.go:111
+msgid "The name for the newly created object."
+msgstr "O nome para o objeto recém criado."
+
+#: pkg/kubectl/cmd/autoscale.go:72
+msgid ""
+"The name for the newly created object. If not specified, the name of the "
+"input resource will be used."
+msgstr ""
+"O nome para o objeto recém criado. Se não especificado, o nome do input "
```

Comment:
Se não especificado, o nome do objeto inserido (input resource here in this context I guess is something like ``kubectl autoscale lalala`` and the "lalala" will be used because you haven't specified the --name=lalala